appengine/flexible/CloudTasks/README.md

```diff
@@ -0,0 +1,53 @@
+# Hello World for Google App Engine Flexible Environment
+
+This sample application is ready to deploy to Google App Engine Flexible Environment.
+
+## Prerequisites
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/dotnet/docs/setup).**
+
+2.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/).  The Google Cloud SDK
+    is required to deploy .NET applications to App Engine.
+
+3.  Install the [.NET Core SDK, version 2.0](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.5-download.md).
+
+
+## ![PowerShell](../.resources/powershell.png) Using PowerShell
+
+### Run Locally
+
+```psm1
+PS > dotnet restore
+PS > dotnet run
+```
+
+### Deploy to App Engine
+
+```psm1
+PS > dotnet restore
+PS > dotnet publish
+PS > gcloud beta app deploy .\bin\Debug\netcoreapp2.1\publish\app.yaml
+```
+
+
+## ![Visual Studio](../.resources/visual-studio.png) Using Visual Studio 2017
+
+Visual Studio is *optional*.  An old, unmaintained branch of samples that work
```

Comment:
Remove the sentence about Visual Studio 2015, because we don't have a CloudTasks sample in the old branch.

Reply:
This has been rewritten - PTAL?

Reply:
Did you push the updated version?

Reply:
Apparently not. Looks like Git ate it. 😢

Reply:
Take two done - PTAL? 😄